src/main/resources/com/puppycrawl/tools/checkstyle/messages.properties

```diff
@@ -1,4 +1,8 @@
 Checker.finishLocalSetup=if no custom moduleFactory is set, moduleClassLoader must be specified
+Checker.processFilesException=Exception was thrown while processing
+Checker.setCharset=unsupported charset
+Checker.setupChildModule=cannot initialize module
+Checker.setupChildNotAllowed=is not allowed as a child in Checker
```

Comment:
All of them should have `{0}` to clearly define that some value will be there

Reply:
Done and rebased....

Hacked the new method `getLocalizedMessage` to take another argument _**Varargs**_. Also, modified the driver codes to take params. There weren't much major changes required here bcoz `getMessage` already calls format on args if varargs is not null, just like our getCheckMessage :)